source/users/custom-metadata.txt

```diff
@@ -712,6 +725,19 @@ Configure Authentication Provider Metadata
          If you successfully configure the provider's metadata
          fields, App Services returns a ``204`` response.
 
+.. note:: Tips for Users with Multiple Linked Authentication Providers  
+
+  - To ensure the most up-to-date metadata, users should re-authenticate after 
+    switching to a different authentication provider. Failure to do so may result 
+    in outdated metadata reflected in the user's table and for requests 
```

Comment:
This is a table with a list of all the users so I don't think this should be possessive!
```suggestion
    in outdated metadata reflected in the Users Table and for requests 
```

Reply:
Do you know whether the table name is ``user`` or ``users``? Would like to rephrase user table to 'user collection', as we don't typically refer to data as being stored in tables in the docs

Reply:
ah when I say "Users Table", I mean an actual table component in our UI that contains a list of users for the admin. But actually, it might be clearer to say the "Users Page" instead if you're good with that!

Reply:
ok makes sense, thank you! Updated the wording, let me know what you think!